src/cmp.rs

```diff
@@ -14,6 +16,7 @@
 
 /// A type with an approximate equivalence relation.
 pub trait ApproxEq {
+    /// The epsilon type used measure an error.
     type Eps: Sized;
```

Comment:
Wondering if we could eventually use https://github.com/brendanzab/approx for this...